apps/src/applab/appStorage.js

```diff
@@ -2,13 +2,62 @@
 
 /* global Applab */
 
-var RecordListener = require('./RecordListener');
+var Firebase = require("firebase");
 
 /**
  * Namespace for app storage.
  */
 var AppStorage = module.exports;
 
+// HACKHACK: Need to make this configurable
+var BASE_DB_URL = 'https://radiant-fire-5518.firebaseio.com';
```

Comment:
The URL isn't the secret.  There is a way to make access to the database require an auth key and also a way to plug in your own custom authentication.  (https://www.firebase.com/docs/web/guide/login/custom.html#section-authenticating-clients)  

I haven't read up on this yet and it isn't included in this experiment;  we would need to study that and in include in the design if we go the firebase route.